src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -120,6 +120,11 @@ configurationRegistry.registerConfiguration({
 			'type': 'boolean',
 			'default': TERMINAL_DEFAULT_RIGHT_CLICK_COPY_PASTE
 		},
+		'terminal.integrated.copyOnSelection': {
+			'description': nls.localize('terminal.integrated.copyOnSelection', "When set, text selected in the terminal will be copied to the clipboard."),
```

Comment:
Not actually sure how localization works in this project. Is anything else needed to properly localize this string?

Reply:
Nope, that's it 😃